samples/macOS/clean/ls.sdiff.level_2

```diff
@@ -1,2 +1,2 @@
-*** changed: macOS/clean/ls
+--- missing: .
```

Comment:
Hmmm, I'll look into this as well.

Reply:
Resolved in [`0b94a09` (#350)](https://github.com/chainguard-dev/bincapz/pull/350/commits/0b94a097a4413b90e77b234b3560fe3c4d2917be).